File: AkitaModel/Results/Note.cs
namespace Skyware.Lis.AkitaModel.Results;

// Ignore Spelling: Dismissible

/// <summary>
/// Represents a note that may appear at visit or department level.
/// </summary>
public class Note : NoteBase
{

    /// <summary>
    /// Level of importance of the note.
    /// </summary>
    public NoteLevels Level { get; set; } = NoteLevels.Info;

    /// <summary>
    /// True if the note can be dismissed by the user.
    /// </summary>
    public bool IsDismissible { get; set; } = false;

}

File: AkitaMotelTS/src/Results/Note.ts
/**
 * Skyware.Lis.AkitaModel.Results
 */

import { NoteBase } from '../NoteBase';
import { NoteLevels } from './NoteLevels';

// Ignore Spelling: Dismissible

/**
 * Represents a note that may appear at visit or department level.
 */
export class Note extends NoteBase {
  /**
   * Level of importance of the note.
   */
  level: NoteLevels = NoteLevels.Info;

  /**
   * True if the note can be dismissed by the user.
   */
  isDismissible: boolean = false;
}
